samples/subsys/ipc/rpmsg_service/sample.yaml

```diff
@@ -11,6 +11,7 @@ tests:
       - mps2_an521
```

Comment:
Should the allow list be updated?

Reply:
What is the allow list in this context?
I still need to re-run this after the holiday break but this test still passed on mps2_an521 provided it was built with sysbuild.